docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3"
+version: '3'
```

Comment:
Unrelated, but I'm 99% sure we're not using this file. There's a Dockerfile instead.

Reply:
Want me to delete the `docker-compose.yml` file then?

Reply:
Sure, although that seems unrelated. I can take care of that in a separate PR as well.

Reply:
I'll let you do that in a separate PR then. 😉